src/main.rs

```diff
@@ -11,6 +11,15 @@ extern crate fnv;
 #[cfg(feature = "sysalloc")]
 extern crate alloc_system;
 
+#[macro_export]
```

Comment:
I personally think that macros are untidy, macros that hide control flow doubly so, and I've been trying to minimize their use.  Looking forward to `?` being stable so we can get rid of `try!`.